### ISSUE TITLE
How to remove classes from pre-trained models (or provide scripts that build the pretrained modes)

### ISSUE THREAD

**Issue by jheaff1** [google-coral/edgetpu#889]
Hi,

Would it be possible to provide the scripts used to build the models provided here - https://coral.ai/models/object-detection/

Specifically, I would like a version of https://github.com/google-coral/test_data/blob/master/ssd_mobilenet_v2_coco_quant_postprocess.tflite that detects only 4 of the COCO classes rather than all 90.

If the script to generate the model were provided, I could rework it as required.

Many thanks!

**Comment by Curid** [comment]
What are you hoping to accomplish by removing classes from the model?

**Comment by jheaff1** [comment]
I’m hoping that the inference would run faster and use less power.

**Comment by Curid** [comment]
I don't think you can remove classes without retraining the entire model, and I doubt It'd be more than 25% faster.

https://coral.ai/docs/edgetpu/retrain-detection https://codeberg.org/Curid/TF-CCTV

**Comment by jheaff1** [comment]
My primary motivation is to reduce the power consumption of the device running the Coral model. If the model could be 25% faster then it may also consume less power.

I’m happy with the accuracy and speed of the Coral model, which is why I was wondering if Google provide scripts to train the models, which I could modify to remove the classes I don’t need.
